util/makesrc

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 
-#    makesrc <tag>
-#    requires ruby gem github_changelog_generator
-#    API_KEY generated here https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token
+#    makesrc <tag> <pat> 
+#    personal access token allows around rate limit
 
-if [ -e $1 ]; then
-	echo "makesrc <tag>" >&2
+if [ -e $2 ]; then
```

Comment:
If we remove the changelog section below, we should probably clean up all the $PAT related items too